cairo/ethereum/crypto/kzg.cairo

```diff
@@ -1,11 +1,16 @@
 from starkware.cairo.common.cairo_builtins import BitwiseBuiltin
+from starkware.cairo.common.bitwise import bitwise_and
+from starkware.cairo.common.math import unsigned_div_rem
+from starkware.cairo.common.math_cmp import is_not_zero
 from ethereum_types.bytes import Bytes32, Bytes
-from ethereum_types.numeric import U256, U256Struct, U384
+from ethereum_types.numeric import U256, U256Struct, U384, bool
 from ethereum.utils.numeric import U256_from_be_bytes32, U256_le, U384_from_be_bytes
 from cairo_ec.curve.bls12_381 import bls12_381
 
 using BLSScalar = U256;
 
+const GET_FLAGS_MASK = 2 ** 95 + 2 ** 94 + 2 ** 93;
```

Comment:
TIL this works lol
perhaps my comment was irrelevant then

Reply:
I suppose it is computed at compile time. Even then it is better for readability